inline-caching/cache-loader/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id 'java'
+}
+
+group = 'com.vmware.gemfire.inlineCaching'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '1.8'
+
+
+task copyAllDependencies(type: Copy) {
+    from configurations.compileClasspath
+    include 'jooq-3.6.2.jar', 'HikariCP-4.0.3.jar', 'postgresql-42.6.0.jar'
```

Comment:
does this magic use exact resolution or could you wildcard it... i.e `jooq*`

Reply:
that'll work. done